client/state/utils/actions.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import { UTILS_UNIQUEID_INCREMENT } from 'state/action-types';
+
+export function uniqueId( prefix ) {
+	return ( dispatch, getState ) => {
+		dispatch( { type: UTILS_UNIQUEID_INCREMENT } );
+		const id = getState().utils.uniqueId + '';
```

Comment:
It's a few more characters, but I think `.toString()` more clearly expresses your intent here, rather than relying on coercion (which is arguably undesirably "clever").